src/Exceptions/HTTPRequestException.php

```diff
@@ -2,6 +2,9 @@
 
 namespace MeiliSearch\Exceptions;
 
+use function array_key_exists;
```

Comment:
Maybe we can add this line into `.php_cs.dist` to get a more stable/concise way of doing it:

```php

$rules = [
    // ...
    'native_function_invocation' => ['include' => ['@compiler_optimized'], 'scope' => 'namespaced'],
]
```
What do you think?

Reply:
What this configuration does imply? 🙂

Reply:
instead of doing the `use function array_key_exists;` and everybody can decide if they should or not do this one we can add it in the configuration and actually it will do `\array_key_exists()`.

The reason for even doing this is because of some performance gains.

An example https://github.com/scssphp/scssphp/issues/96 measurement done by a symfony maintainer.

Reply:
So interesting! Thanks @devrck for the explanation!

> Of course at big scale you feel it but in the day to day you won't even notice it.

Completely agree. If we can easily do improvement now, we take it!

Reply:
Of course we can open a new PR and apply it in the whole codebase instead of putting more in this PR.

Reply:
@devrck
Are there other native functions we need to change in this package (by putting the `\` as prefix name)?
In this case, we need to do a new PR, but I just wanted to know 🙂

Reply:
Here is one example https://github.com/meilisearch/meilisearch-php/blob/master/src/Index.php#L282 that is outside of @emulienfou PR and I think it should be done separate.

This is the full list https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/2.16/src/Fixer/FunctionNotation/NativeFunctionInvocationFixer.php#L357

Reply:
@devrck
Are you ok to do a complete PR to add this improvement? 🙂
I'm going to merge this PR without.
If you can I will do it anyway when I have time 😉

Reply:
Sure thing.

Reply:
I've just created an issue #53